### PR TITLE
Bump dependencies on 2020-10-07

### PR DESCRIPTION
**Pull request by AntoineAugusti**
See Trello card https://trello.com/b/y1mE9CZI/gc-notify-sprint-backlog

Bump dependencies and fixed a few coding style errors picked up by the upgraded flake8

### CODE REVIEW

File: requirements_for_test.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-phonenumbers==8.10.13
-pytest==5.1.2
-pytest-mock==1.10.4
-pytest-cov==2.7.1
-pytest-xdist==1.29.0
-requests-mock==1.7.0
-freezegun==0.3.12
-flake8==3.7.8
-flake8-print==3.1.0
+pytest==5.4.3
```
Comment (brdunfield) [inline requirements_for_test.txt:2]: is phonenumbers not needed for tests?

Comment (AntoineAugusti) [reply requirements_for_test.txt:2]: It's already listed in `setup.py` so I removed it here. Sorry, I should have added a note!

**Comment by brdunfield** [comment]
We will need a version bump for utils here :)

**Comment by AntoineAugusti** [comment]
Right, I should have known, I've made several releases of utils already 🙃  Thanks!

**Review by brdunfield — APPROVED** [review]
🦑 Sorry for the delay in approving it - wanted to test locally and had meetings. Works great!

Can we add this update to the existing Admin PR once merged? I will review that next! :)

**Comment by AntoineAugusti** [comment]
No worries! Bumped utils on admin + API right now

**Pull request merged**